Scripts/Editor/Core/CollectionItemIndirectReferencePropertyDrawer.cs

```diff
@@ -119,14 +152,15 @@ private Type GetGenericItemType(Type targetType)
             return null;
         }
 
-        private CollectionItemEditorOptionsAttribute GetOptionsAttribute()
+        private SOCItemEditorOptionsAttribute GetOptionsAttribute()
         {
             if (fieldInfo == null)
                 return null;
-            object[] attributes = fieldInfo.GetCustomAttributes(typeof(CollectionItemEditorOptionsAttribute), false);
+            object[] attributes = fieldInfo.GetCustomAttributes(typeof(SOCItemEditorOptionsAttribute), false);
             if (attributes.Length > 0)
-                return attributes[0] as CollectionItemEditorOptionsAttribute;
-            return null;
+                return attributes[0] as SOCItemEditorOptionsAttribute;
+
+            return new SOCItemEditorOptionsAttribute();
```

Comment:
Why does this return a new attribute instance instead of `null` now?

It's not intuitive to me that it's looking for an existing attribute and if it doesn't exist, it returns one regardless. If there's a good reason for it then it might be worth specifying what that is.

Reply:
So instead of returning null, I just want to return a new one that would be treated as a default one, but yeah this is misleading and even worst could be happening inside custom property drawers multiple times per frame, so yeah I will fix it!